### ISSUE TITLE
Can't compile / install due to missing ZIP binary on Windows

### ISSUE THREAD

**Issue by d0k3** [Steveice10/citrus#14]
As the title says... with previous versions of this, I could just comment that line out and it still worked, but now ZIP.exe seems to be a absolute requirement for being able to install. Maybe you could provide a link for Windows users on how to get ZIP.exe?

**Comment by Steveice10** [comment]
I don't have much experience with building on Windows, but if you don't have zip you can always just comment out the lines that use it from the buildtools Makefile base.

**Comment by d0k3** [comment]
Well, I know about this issue. I've been researching some more, and stumbled upon this: http://www.willus.com/archive/zip64/
These are compiled binaries of info-zip, which works as a swap-in replacement. Source code is found here:
https://sourceforge.net/projects/infozip/

Commenting out the lines does not do it, you need to change a lot more for install to work again, because as install works now it zips, then unzips.

And, as you said, this is actually an issue with buildtools (on Windows, ofc), which is used by other projects as submodule (I'm looking to include it with CTRXplorer, f.e.), too. So, maybe add a link to the info-zip download to the description?

**Comment by damnnfo** [comment]
I had the same issue on my home PC before. 
So I tried it again on my office PC yesterday. And it works flawlessly.
Then I tried to find out what's the difference between those two PCs.
Finally I found out that Oracle client 11.2.0 (which includes "zip.exe" and "unzip.exe" in its own bin folder) was installed on my office PC. And fortunately it's already added to Windows Path.

C:\devkitPro\citrus\tools>zip
Copyright (c) 1990-2008 Info-ZIP - Type 'zip "-L"' for software license.
Zip 3.0 (July 5th 2008).
